docs/kubernetes/domains-google-cloud-infrastructure.md

```diff
@@ -0,0 +1,53 @@
+# Domains in Google Cloud infrastructure.
+Every change in domain needs to be done manually. To pass traffic through domain name you need to modify `ingress.yml` and `webserver-php-fpm.yml` which serves as webserver and listens on domain names.
+
+## Adding Domain
+Let's say we want to listen on new domain name.
+
+Open [ingress.yml](/project-base/kubernetes/ingress.yml) file and add new domain block into `spec -> rules`:
+
+```
+// kubernetes/ingress.yml
+
+rules:
+    -   host: ~
+        http:
+            paths:
+            -   path: /
+                backend:
+                    serviceName: webserver-php-fpm
+                    servicePort: 8080
+```
+*Note*: ${DOMAIN_INDEX} is a index of domain in ingress.yml
+
+Create secrets from TSL certificate in `ingress-patch.yml` in `kubernetes/kustomize/overlays/production/ingress-patch.yml`:
+```
+// kubernetes/kustomize/overlays/production/ingress-patch.yml
+
+- name: domain-${DOMAIN_NUMBER}-ssl-certificates
+  commands:
+    tls.key: "cat ${ANOTHER_DOMAIN_SSL_DIRECTORY}/tls.key"
+    tls.crt: "cat ${ANOTHER_DOMAIN_SSL_DIRECTORY}/tls.crt"
+    ca.crt: "cat ${ANOTHER_DOMAIN_SSL_DIRECTORY}/ca.crt"
+```
+
+Next, add TSL certificates for new domain:
+
+```
+spec:
+    tls:
+    -   hosts:
+        secretName: domain-${DOMAIN_NUMBER}-ssl-certificates
+```
+
+Open [.ci/deploy-to-google-cloud.sh](/project-base/.ci/deploy-to-google-cloud.sh) file and set your new domain host to `ingress.yml` and `webserver-php-fpm.yml` host name:
+
+```
+// .ci/deploy-to-google-cloud.sh
+
+NEW_DOMAIN_HOST=${NEW_DOMAIN_HOST}
+yq write --inplace kubernetes/ingress.yml spec.rules[${DOMAIN_INDEX}].host ${NEW_DOMAIN_NAME}
+yq write --inplace kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.hostAliases[0].hostnames[+] ${NEW_DOMAIN_NAME}
+```
+
+Dont forget to pass `NEW_DOMAIN_HOST` and `ANOTHER_DOMAIN_SSL_DIRECTORY` to `shopsys/kubernetes-buildpack` as ENV variable.
```

Comment:
~~Dont~~ -> Do not.

To be honest, I am missing some conclusion here. Like what is done now? After performing all the described steps, I have a new domain added to my Google Cloud infrastructure?